tests/administration/realms.test.js

```diff
@@ -1,7 +1,54 @@
+'use strict';
 const common = require('../common'),
       expect = require('chai').expect;
 
 describe('Realms', () => {
   before(() => common.login(browser, 'admin', 'admin', 'local'));
+  after(() => common.logout());
   it('should navigate to Realms', () => common.clickSidebarTab(browser, 'Realms'));
+
+  it('should create a valid realm', () => {
+  	return common.waitForOnClick('//*[@id="add_realm"]')
+  		.then(() => browser.click('//*[@id="add_realm"]'))
+  		.then(() => browser.waitUntil( browser.getAttribute('//*[@id="realm"]','onchange')
+  			.then((onc) => onc !== null )
+  		))
+  		.then(() => browser.setValue('//*[@id="realm"]', 'test'))
```

Comment:
If the intent is to set a valid realm, then this should be HIVEIO - as we changed the page to indicate that the value needs to be the NetBIOS name of the domain.

Granted 'test' will pass the validation, but had we had validation of the NetBIOS name, it should have failed.